docs/networks/mainnet/running-a-node.md

```diff
@@ -104,20 +104,7 @@ If you want more convenience and your validator to operate quickly, you can also
 
 > After the synchronization is finalized, you will end up with the equal blockchain data. You can use the flag on every startup. However, it shows the most significant effect when synchronizing from scratch or after an extended downtime. The shortcut is ideal for fresh installations, validator migration, or recovery.
 
-<Tabs>
```

Comment:
Why removing this? Once the checkpoint sync is not down anymore, this section will still be useful to have no?

Reply:
its not removed- the order of the menu bar changed (its added down below). checkpoint sync should stay as first option